### PR TITLE
Add instructions on running tests to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tedkornish**

### CODE REVIEW

**Comment by begriffs** [comment]
Good idea, thanks.

**Comment by begriffs** [comment]
Actually, what do you think about linking to the docs in contributing.md instead? https://postgrest.readthedocs.io/en/v0.4/install.html#postgrest-test-suite

**Comment by tedkornish** [comment]
Sounds like a great idea. Will diff back up later today.

**Comment by begriffs** [comment]
Still interested in making the change?

**Comment by tedkornish** [comment]
Indeed; sorry for the delay.

**Comment by begriffs** [comment]
Oh no problem, I'm grateful for your contribution.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -61,3 +61,9 @@ my other obligations.  If you don't get a response right away
 don't worry, I will definitely get to it. Also you can join the
 Gitter [chat room](https://gitter.im/begriffs/postgrest) to
 discuss issues you are having.
+
+## Running Tests
+
+For instructions on running tests, see the official docs hosted here:
+
+https://postgrest.readthedocs.io/en/v0.4/install.html#postgrest-test-suite
```
Comment (begriffs) [inline CONTRIBUTING.md]: There's actually a handy domain alias: https://postgrest.com/en/v0.4/install.html#postgrest-test-suite

**Pull request merged**